docs2/site/sitemap.yml

```diff
@@ -114,6 +114,8 @@
             file: gql010.md
           - title: GQL011
             file: gql011.md
+          - title: GQL012
+            file: gql012.md
```

Comment:
Yeah I think i'd just remove GQL005 from the site map if it is being removed in this version.  This is in the develop branch so the docs won't publish btw.

Reply:
But if someone will use an older version after v8 release, he won't be able to navigate to the GQL005 help link. I'd prefer to keep the old documentation.

Reply:
Fair enough.